### ISSUE TITLE
fsLR parcellations

### ISSUE THREAD

**Issue by effigies** [templateflow/templateflow#102]
Is there anything stopping us from providing parcellations on fsLR (e.g. https://balsa.wustl.edu/study/WG33) directly in templateflow?

**Comment by oesteban** [comment]
I don't think so. However, TemplateFlow needs to have a better representation and querying of atlases (#101) so we might want to address that limitation first, in case we decide to substantially change how atlases are stored (esp. metadata).
